README.md

```diff
@@ -245,3 +245,12 @@ flags:
 ```
 
 This will remap the "environment" Terraform variable to `--env` on the astro command line. You can also specify a description that will show up in the `--help` text.
+
+## Contributing to astro
```

Comment:
I would change this heading to "Developing"

Reply:
I do think that this belongs to the wiki. I'll try to move it there.
I agree that readme should be short and "how to make the release" doesn't really belong there.